updatedb.py

```diff
@@ -1,34 +0,0 @@
-import os
```

Comment:
keep updatedb.py
move it to a separate folder if you'd like
it's useful when making changes to db.sqlite3

Reply:
Yea so we can use db browser directly to make changes to both table and schema

Reply:
much better than coding

Reply:
And you can also run sql code directly in db browser if we still need to